code/modules/research/designs/medical_designs.dm

```diff
@@ -84,6 +84,39 @@
 	build_path = /obj/item/weapon/reagent_containers/glass/beaker/noreact
 	category = list("Medical Designs")
 
+/datum/design/bluespacesyringe
+	name = "Bluespace Syringe"
+	desc = "An advanced syringe that can hold 60 units of chemicals"
+	id = "bluespacesyringe"
+	req_tech = list("bluespace" = 1, "materials" = 2)
```

Comment:
I thought of reducing it a bit below the beaker because the blue beaker is a big game changer to chem, considering you can make 600u grenades with them. what do you suggest I increase, the